src/audio/style.scss

```diff
@@ -11,6 +14,15 @@
 		margin-top: 0.5em;
 	}
 
+	vinyl-controller {
+		--media-background-color: var(--vinyl-background-color);
+		--media-control-background: var(--vinyl-background-color);
+		--media-control-hover-background: transparent;
+
+		/* The primary color is used for --media-text-color and --media-icon-color. */
+		--media-primary-color: var(--vinyl-text-color);
```

Comment:
This accomplishes a customizable background and foreground, though there are a lot of other custom properties associated with hover states and the track and thumb that need to be customizable.